src/html/snis/form.html

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+    <h3 class="header">{{title}}</h3>
+
+    <div class="row">
+        <div class="col s5">
+            <div class="card-panel teal">
+                <p class="white-text">
+                    The SNI object of Kong
+                </p>
+
+                <p class="white-text">
+                    ...
+                </p>
+                <p class="white-text">
+                    Checkout <a href="https://getkong.org/docs/admin-api/#add-sni" target="_blank">Kong documentation</a> for the meaning of these parameters.
```

Comment:
This page doesn't exist on getkong.org. Should be https://getkong.org/docs/latest/admin-api/#add-sni